src/main/java/spoon/experimental/modelobs/FineModelChangeListener.java

```diff
@@ -51,6 +52,8 @@
 
 	void onSetAdd(CtElement currentElement, CtRole role, Set field, ModifierKind newValue);
 
+	void onSetAdd(CtElement currentElement, CtRole role, Set field, CtModuleRequirement.RequiresModifier newValue);
```

Comment:
Are we really going to have copy of this method (and others) for each non CtElement type?